cfme/common/provider_views.py

```diff
@@ -542,7 +542,7 @@ def before_fill(self, values):
         for widget_name in self.COND_WIDGETS:
             widget = getattr(self, widget_name)
             if widget.is_displayed:
-                widget.fill(values.get(widget_name))
+                widget.fill(values.get(widget_name, 'Disabled'))
 
```

Comment:
**REQUIRED:** This will fill ```['prov_type', 'metrics_type', 'alerts_type', 'virt_type']``` with 'Disabled' now instead of, I'm assuming 'None'. Is this the right thing to do for ALL the widgets?